app/backend/prisma/schema.prisma

```diff
@@ -1,80 +1,64 @@
-// This is your Prisma schema file,
```

Comment:
It looks like every line here has been changed.
Can you fix the diff again?

Reply:
this only changes with lines positions, Do you mean get back lines positions ?

Reply:
we sould use common settings when formatting files. Can you remove the format changes. We can consider on friday to use some prettier/prisma plugin to have common settings, but let's not change it in this pull request.